lib/util/filesystem.ml

```diff
@@ -103,19 +103,20 @@ let walk_folder ?(recursive = false) f path acc =
   in
   traverse [ path ] acc
 
-let copy_file ?(perm = 0o640) src dst =
+let copy_file ?(perm = 0o644) src dst =
```

Comment:
I think that we should keep the previous default and fix the documentation. `0o640` is the common default for files served by web servers.